.github/workflows/scripts/install.sh

```diff
@@ -97,8 +86,3 @@ df -hl
 cat /proc/cpuinfo
 sudo apt install virt-what
 sudo virt-what
-
-# For the source tests
-cd ..
-git clone https://github.com/pulp/pulpcore
-git clone https://github.com/pulp/pulp_file
```

Comment:
What's the benefit of moving these lines?

Reply:
avoiding sed delete on https://github.com/pulp/plugin_template/pull/345#discussion_r584564298

Reply:
Ah, now i see. That was mine. ;)